sequencing/ATACSeq/jobSubmission/2_batchCalcQCMetrics.sh

```diff
@@ -6,46 +6,68 @@
 #SBATCH --nodes=1 # specify number of nodes.
 #SBATCH --ntasks-per-node=16 # specify number of processors per node
 #SBATCH --mail-type=END # send email at job completion 
-#SBATCH --output=ATACSeq/logFiles/%u/calcATACQC-%A_%a.o
-#SBATCH --error=ATACSeq/logFiles/%u/calcATACQC-%A_%a.e
-#SBATCH --job-name=calcATACQC
+#SBATCH --output=ATACSeq/logFiles/%u/ATACcalcQCS2-%A_%a.log
+#SBATCH --error=ATACSeq/logFiles/%u/ATACcalcQCS2-%A_%a.err
+#SBATCH --job-name=ATACcalcQCS2
 
-#-----------------------------------------------------------------------#
+## ===================================================================================================================##
+##                             ATAC-seq pipeline STEP 2: Post alignment processing                                    ##
+## ===================================================================================================================##
+## PROOFREADER : Marina Flores-Payan                                                                                  ||
+## CONTACT: m.flores-payan@exeter.ac.uk                                                                               ||
+## LAST UPDATE: February 2024                                                                                         ||
```

Comment:
As per @ejh243 last comment, we expect to remove these lines as they can be found through git blame *etc.*